src/app/components/calendar/calendarForm.tsx

```diff
@@ -36,17 +32,18 @@ const CalendarForm = () => {
   const [calendarSetup, setCalendar] = useLocalStorage('calendarData', calendarDefaultValues)
   const [calendarDates, setCalendarDates] = useLocalStorage('calendarDatesData', calendarDatesDefault)
 
+  // https://dev.to/gvegacl/nextjs-server-prop-did-not-match-client-prop-and-conditional-styleclass-rendering-2i4c
   const [mount, setMount]= useState(false)
 
   // TMP
-  const handleInput = (e: { target: { [key: string]: any }}) => {
-    const fieldName = e.target.name;
-    const fieldValue = e.target.value;
+  const handleInput = (e: React.ChangeEvent<Element>) => {
```

Comment:
This should give target the right type:

```suggestion
  const handleInput = (e: React.ChangeEvent<HTMLInputElement>) => {
```